demo-simple/build.gradle

```diff
@@ -1,13 +1,13 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 27
-    buildToolsVersion "27.0.0"
+    compileSdkVersion 26
```

Comment:
These need to be reverted to the latest version before releasing the library, left at 26 for now so we can have access to the latest docs in the code.